src/nvim/api/private/helpers.c

```diff
@@ -820,6 +820,12 @@ void modify_keymap(Buffer buffer, bool is_unmap, String mode, String lhs,
   }
   buf_T *target_buf = find_buffer_by_handle(buffer, err);
 
+  if (!target_buf) {
+    api_set_error(err, kErrorTypeValidation,
```

Comment:
not needed, `find_buffer_by_handle` already does this.

Reply:
Done